mmv1/products/blockchain/BlockchainNodes.yaml

```
@@ -0,0 +1,145 @@
+        name: endpointInfo
+        output: true
+        description: |
+          The endpoint information through which to interact with a blockchain node.
+        properties:
+          - !ruby/object:Api::Type::String
+            name: jsonRpcApiEndpoint
+            output: true
+            description: |
+              The assigned URL for the node JSON-RPC API endpoint.
+          - !ruby/object:Api::Type::String
+            name: websocketsApiEndpoint
+            output: true
+            description: |
+              The assigned URL for the node WebSockets API endpoint.
+  - !ruby/object:Api::Type::NestedObject
+    name: ethereumDetails
+    description: |
+      User-provided key-value pairs
+    properties:
+      - !ruby/object:Api::Type::NestedObject
+        name: gethDetails
+        description: |
+          User-provided key-value pairs
+        properties:
+          - !ruby/object:Api::Type::Enum
+            name: garbageCollectionMode
+            immutable: true
+            description: |
+              Blockchain garbage collection mode.
+      - !ruby/object:Api::Type::NestedObject
+        name: additionalEndpoints
+        description: |
+          User-provided key-value pairs
+        properties:
+          - !ruby/object:Api::Type::String
+            name: beaconApiEndpoint
+            output: true
+            description: |
+              The assigned URL for the node's Beacon API endpoint.
+          - !ruby/object:Api::Type::String
+            name: beaconPrometheusMetricsApiEndpoint
+            output: true
+            description: |
+              The assigned URL for the node's Beacon Prometheus metrics endpoint.
+          - !ruby/object:Api::Type::String
+            name: executionClientPrometheusMetricsApiEndpoint
+            output: true
+            description: |
+              The assigned URL for the node's execution client's Prometheus metrics endpoint.
+      - !ruby/object:Api::Type::Enum
+        name: network
+        immutable: true
+        description: |
+          The Ethereum environment being accessed.
+      - !ruby/object:Api::Type::Enum
+        name: executionClient
+        immutable: true
+        description: |
+          The execution client
+      - !ruby/object:Api::Type::Enum
+        name: consensusClient
+        immutable: true
+        description: |
+          The consensus client
+      - !ruby/object:Api::Type::Boolean
+        name: apiEnableAdmin
+        immutable: true
+        description: |
+          Enables JSON-RPC access to functions in the admin namespace. Defaults to false.
+      - !ruby/object:Api::Type::Boolean
+        name: apiEnableDebug
+        immutable: true
+        description: |
+          Enables JSON-RPC access to functions in the debug namespace. Defaults to false.
+
+  - !ruby/object:Api::Type::Enum
+    name: blockchainType
+    immutable: true
+    description: |
+      User-provided key-value pairs
```

```suggestion
  - !ruby/object:Api::Type::Enum
    name: blockchainType
    immutable: true
    description: |
      User-provided key-value pairs
```

Another enum! Also please remove the extra whitespace